.github/actions/install-crucible/install-crucible.sh

```diff
@@ -72,13 +72,30 @@ done
 validate_ci_run_environment
 validate_ci_endpoint
 
+ci_auth_file="/root/crucible-ci-engines-token.json"
+production_auth_file="/root/crucible-production-engines-token.json"
+if [ -e "${ci_auth_file}" -a -s "${ci_auth_file}" -a -e "${production_auth_file}" -a -s "${production_auth_file}" ]; then
```

Comment:
I still don't understand the "-s" to read from stdin here...

Reply:
`-s` is just a test for a non-empty file